grid/app/main/routes/dfl/routes.py

```diff
@@ -24,16 +25,16 @@
 # ======= WEB ROUTES ======
 
 
-@main.route("/favicon.ico")
+@dynamic.route("/favicon.ico")
 def favicon():
     return send_from_directory(
-        os.path.join(main.root_path, "static"),
+        os.path.join(dynamic.root_path, "static"),
```

Comment:
What is this for? Is it necessary?

Reply:
It's the path for static files. I will go ahead and add it directly to the Blueprint.

Reply:
See my comment reply below.